.config/ansible-lint.yml

```diff
@@ -3,7 +3,6 @@ skip_list:
   - command-instead-of-module # Using command rather than module where necessary
   - command-instead-of-shell # Using command rather than shell where necessary
   - experimental # all rules tagged as experimental
-  - fqcn[action]
```

Comment:
I have been trying for a long time to ensure compatibility with older versions of ansible, but it seems it's time to raise the minimum version to 2.11, for example, because now there is no compatibility with older versions.

Please mark the minimum version in `minimal_ansible_version` variable and in the README.

Reply:
Done, pull request is ready